README.md

```diff
@@ -11,17 +11,17 @@ Broadside offers a simple command-line interface to perform deployments on ECS.
 
 ```ruby
 Broadside.configure do |config|
-  config.base.application = 'hello_world'
-  config.base.docker_image = 'lumoslabs/hello_world'
-  config.deploy.type = 'ecs'
+  config.application = 'hello_world'
+  config.docker_image = 'lumoslabs/hello_world'
```

Comment:
if we ever think we will want to be able to configure `docker_image` on a per target basis, now would be the time to do it.  i left it how it was.